### PR TITLE
docs(de): update contributing file

### PR DESCRIPTION
**Pull request by SoftCreatR**
## Checklist

- [x] I ran `npm test` locally and it passed without errors.

See #1610

### CODE REVIEW

File: docs/de/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 
 Für beide Szenarien ist der Aufbau gleich, jedoch unterscheiden sich die Beitragsschritte geringfügig.
 
 ## Ich habe noch nie zu Open Source beigetragen
 
 Einen Beitrag zu Open Source zu leisten, kann zunächst einschüchternd sein. Aus diesem Grund haben wir eine
-Neue Mission in [TwilioQuest](https://www.twilio.com/quest), die dich
-Schritt für Schritt durch das Erstellen eines Beitrags für dieses Projekt führt. Sobald Du [das Spiel heruntergeladen](https://www.twilio.com/quest/download) hast, kannst Du die Mission auswählen und Du wirst durch alle Schritte durchgeführt, vom Klonen des Projekts bis zur Erstellung deines Pull-Requests.
+neue Mission in [TwilioQuest](https://www.twilio.com/quest), die dich
+Schritt für Schritt durch das Erstellen eines Beitrags für dieses Projekt führt. Sobald Du [das Spiel heruntergeladen](https://www.twilio.com/quest/download) hast, kannst du die Mission auswählen und wirst anschließend durch alle Schritte durchgeführt, vom Klonen des Projekts bis zur Erstellung deines Pull-Requests.
```
Comment (dkundel) [inline docs/de/CONTRIBUTING.md]: There's another `Du` in this line. Could you fix that one as well?

**Comment by dkundel** [comment]
Thanks for the contribution :) I found one more typo, it would be great if you could fix that one and I'm happy to merge your PR :) thank you!

**Comment by SoftCreatR** [comment]
Sure, will do that asap :)

**Comment by SoftCreatR** [comment]
Done, with some other, minimal changes/optimizations.

**Comment by SoftCreatR** [comment]
@dkundel 👋

**Comment by SoftCreatR** [comment]
@dkundel Just curious: Shouldn't i be automatically added to the contributors list?

**Pull request merged**